### PR TITLE
Feature/2781 add link to client

### PR DESCRIPTION
**Pull request by rajneeshweb**
Targets #2781 

### Description

Added link to client detail page on the client name in project details page using route() function

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     },
     "devDependencies": {
         "bootstrap": "^4.4.1",
-        "cross-env": "^7.0.2",
+        "cross-env": "^7.0.3",
```
Comment (GauravGusain98) [inline package.json:18]: Why this is updated? In your file changes, only the files related to your task should be present. Please remove other files like package.json, package-lock.json, and composer.lock

File: Modules/Project/Resources/views/show.blade.php [hunk 2]
```diff
@@ -37,7 +37,7 @@
                         <h4 class="d-inline-block">
                             <label for="name" class="font-weight-bold mb-6 ml-1">Client:</label>
                         </h4>
-                        <span class="text-capitalize ml-2 fz-lg-22">{{ $project->client->name }}</span>
+                        <a href="/client/{{ $project->client->id }}/edit" class="text-capitalize ml-2 fz-lg-22">{{ $project->client->name }}</a>
```
Comment (GauravGusain98) [inline Modules/Project/Resources/views/show.blade.php:40]: We don't hardcode the URL instead use route() function to get the URL. Read about route in Laravel

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by rajneeshweb** [comment]
Closing this branch.

**Pull request closed without merging**